test/e2e/disks.e2e.ts

```diff
@@ -80,20 +80,26 @@ test.describe('Disk create', () => {
 
   test('from snapshot', async ({ page }) => {
     await page.getByRole('radio', { name: 'Snapshot' }).click()
-    await page.getByRole('button', { name: 'Source snapshot' }).click()
+    await page
+      .getByPlaceholder('Select a snapshot or enter a snapshot name', { exact: true })
+      .click()
     await page.getByRole('option', { name: 'delete-500' }).click()
   })
 
   // max-size snapshot required a fix
   test('from max-size snapshot', async ({ page }) => {
     await page.getByRole('radio', { name: 'Snapshot' }).click()
-    await page.getByRole('button', { name: 'Source snapshot' }).click()
```

Comment:
I think you could have just changed `button` to `combobox` on all these selectors and it still works. I can see an argument for selecting by placeholder, and we do it occasionally, but I prefer the field label.